targets/runtime.project.json

```diff
@@ -25,6 +26,7 @@
         "Microsoft.NETCore.Runtime": "1.0.1-beta-23405",
         "Microsoft.NETCore.TestHost": "1.0.0-beta-23318",
         "xunit": "2.1.0",
+        "Newtonsoft.Json": "7.0.1",
```

Comment:
It's a dependency of Microsoft.DotNet.Build.Tasks.dll, which comes from the build tools package.  But it shouldn't be in this PR, so I've rebased it out.